### DISCUSSION TITLE
Additional Renamer

### DISCUSSION THREAD

**Discussion by James-MF-Thomas** [UltimaHoarder/UltimaScraper#649] [Q&A] [answered]
A 2nd naming option to be able to distinguish between paid and free content....? i.e.:

"free_filename_format": "{username}-{media_id}-public.{ext}",
"paid_filename_format": "{username}-{media_id}.{ext}",

....and maybe the ability to enter a custom string?

**Comment by UltimaHoarder** [comment, marked as answer]
????

Read the readme on the formats.

**Reply by James-MF-Thomas** [reply]
Ah, I guess you add {value} the name. 
Though that would put it on both variables instead of just specifying one. 
I've never tried to enter a custom string along with the identifiers, just assumed it wasn't possible.
